versioned_docs/version-1.10/self-hosted/administration/configuration.mdx

```diff
@@ -950,8 +950,8 @@ namespace:
 Configures [network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) for each namespace to isolate network traffic.
 Disabled by default.
 
-- `ingress`: Ingress list of rules ([NetworkPolicyIngressRule](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#networkpolicyingressrule-v1-networking-k8s-io)) to be applied on every namespace managed by Okteto. (optional).
-- `egress`: Egress list of rules ([NetworkPolicyEgressRule](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#networkpolicyegressrule-v1-networking-k8s-io)) to be applied on every namespace managed by Okteto (optional).
+- `ingress`: Ingress list of rules (NetworkPolicyIngressRule) to be applied on every namespace managed by Okteto. (optional).
```

Comment:
```suggestion
- `ingress`: Ingress list of rules (same syntax as `NetworkPolicyIngressRule`) to be applied on every namespace managed by Okteto. (optional).
```
I would add a short explanation here. Or maybe a sample? The intention is that `ingress` is a list of elements, and each element is of the type `NetworkPolicyIngressRule`